core/commands/files/files.go

```diff
@@ -613,6 +617,97 @@ Examples:
 	},
 }
 
+var FilesPublishCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "publish a path to a specified ipns entry",
```

Comment:
yiiiiiikes, no this is a command that should wrap this. recall the massive argument we had over this. this belongs in a higher level construct, what you call mfs. what this is not. (files/ here should be raw manipulations of objects only).

instead, this should work:

```sh
ipfs name publish $name files/path/to/file
# or
ipfs name publish $name /files/path/to/file
```